wrangler-transform/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,16 @@
+#json file path
+Directive_parse_csv=testData/Wrangler\
+  /parse_csv_wrangle-cdap-data-pipeline.json
+
+bqSourceTable=dummy
+sourcePath=example/hello.csv
+#bq queries file path
+CreateBQTableQueryFile=BQtesdata/BigQuery/BigQueryCreateTableQuery.txt
```

Comment:
Note: this logic of create table and importing values works for the current single feature. Once more pipelines are added, we have to change this to account for the different data that we will use.

Reply:
yes  will change accordingly